client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -22,6 +23,15 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 				hasDomainCredit={ plan && plan.hasDomainCredit }
 			/>
 
+			<PurchaseDetail
+				icon="help"
+				title={ i18n.translate( 'Attend a live course' ) }
+				description={ i18n.translate( 'Register for one of our live courses led by Happiness Engineers ' +
+					'to get the most out of your site.'
+				) }
+				buttonText={ i18n.translate( 'Register for a course' ) }
+				href={ support.CALYPSO_COURSES } />
```

Comment:
I wonder if we should add any tracks pings here. I didn't see any for the other purchase details but maybe since we're tracking course clicks from /help we can do it here also.